python/lsst/obs/base/_fitsRawFormatterBase.py

```diff
@@ -101,6 +102,16 @@ def filterDefinitions(self):
         """
         return None
 
+    @property
+    @cached_getter
+    def checked_parameters(self):
+        # Docstring inherited.
+        parameters = super().checked_parameters
+        if "bbox" in parameters:
+            raise TypeError("Raw formatters do not support reading arbitrary subimages, as some "
+                            "implementations may be assembled on-the-fly.")
```

Comment:
I'm not sure this is the right way to disable a storage class's parameter in a particular formatter, because there's also that `unsupportedParameters` class attribute.

But IIRC, that attribute is actually used by assembly/disassembly to know whether a component _StorageClass_ accepts parameters passed when loading the parent, and hence I shouldn't actually remove `bbox` (and `origin`) there.  If that's correct, maybe we should remove that attribute from the base `Formatter` and have that logic look at component StorageClass components instead?  Is there some subtle reason why we can't do that either?